README.rst

```diff
@@ -1274,6 +1274,100 @@ This produces:
 
         CREATE TABLE "names" AS (SELECT "last_name","first_name" FROM "person")
 
+Managing Table Indices
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create Indices
+""""""""""""""""
+
+The entry point for creating tables is ``pypika.Query.create_index``.
```

Comment:
I think you meant "entry point for creating tables"

Reply:
Yeah, copy-paste.